modules/virt-about-vmis.adoc

```diff
@@ -23,3 +23,10 @@ When you delete a VM, the associated VMI is automatically deleted. You delete a
 ====
 Before you uninstall {VirtProductName}, list and view the standalone VMIs by using the CLI or the web console. Then, delete any outstanding VMIs.
 ====
+
+When you edit a VM, some settings may be applied to VMIs dynamically and without the need for a restart. Any change made to a VM object that cannot be applied to the VMIs dynamically will trigger the RestartRequired VM condition.
+
+[NOTE]
+====
+Once the RestartRequired condition is triggered on a VM object, no further changes can be applied. Changes are effective on the next reboot, and the condition is removed.
```

Comment:
`no further changes can be applied` is confusing, it does not prevent to make new changes even the RestartRequired condition is met.